Orange/preprocess/discretize.py

```diff
@@ -98,6 +98,14 @@ def __call__(self, data, variable):
             "Subclasses of 'Discretization' need to implement "
             "the call operator")
 
+    def __repr__(self):
+        args = self.__class__.__init__.__code__.co_varnames
+        return "{}({})".format(
+            self.__class__.__name__,
+            ", ".join("{}={}".format(arg, repr(getattr(self, arg))) for i, arg in enumerate(args) if
+                arg != "self" and self.__class__.__init__.__defaults__[i-1] != getattr(self, arg))
+        )
```

Comment:
Put this in a separate class:
```py
class Reprable:
    def __repr__(self):
        """ Docstring explaining what happens """
        ...
```
Then you can have `Discretization` and `Preprocess` (below) inherit this new type?